_CodingChallenges/031-flappybird.md

```diff
@@ -55,5 +55,9 @@ contributions:
     author: "marvszl"
     url: "https://editor.p5js.org/szloszjar.marvin@gmail.com/full/jE8DQY1Zd"
     source: "https://editor.p5js.org/szloszjar.marvin@gmail.com/sketches/jE8DQY1Zd"
+  - title: "Flappy bird"
+    author: "Syon"
+    url: "ikittycode.github.io/flappy-bird"
+    source: "https://github.com/iKittyCode/snake-game"
```

Comment:
```suggestion
    source: "https://github.com/iKittyCode/flappy-bird"
```
Wanted to check about the source code link. Looks like you might have accidentally linked to your snake game. I think this might have been the link you wanted. If not let me know.

Reply:
Yeah that was the link I wanted